cedar-lean/lakefile.lean

```diff
@@ -18,9 +18,9 @@ import Lake
 open Lake DSL
 
 meta if get_config? env = some "dev" then -- dev is so not everyone has to build it
-require «doc-gen4» from git "https://github.com/leanprover/doc-gen4" @ "c7f4ac84b973b6efd8f24ba2b006cad1b32c9c53"
+require «doc-gen4» from git "https://github.com/leanprover/doc-gen4" @ "v4.10.0"
 
-require "leanprover-community" / "batteries"
+require «batteries» from git "https://github.com/leanprover-community/batteries" @ "v4.10.0"
```

Comment:
any way we can avoid this reversion to the old syntax? see https://lean-lang.org/blog/2024-8-1-lean-4100/ for more on the new syntax and why it's better

Reply:
Updated in the latest commit. Note that we still need to pin to a specific git version, but I _think_ the new syntax is still going through Reservoir.